ngafid-data-processor/src/main/java/org/ngafid/processor/format/ParquetFlightBuilder.java

```diff
@@ -0,0 +1,49 @@
+package org.ngafid.processor.format;
+
+import org.ngafid.core.flights.*;
+import org.ngafid.processor.steps.*;
+
+import java.sql.Connection;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+import static org.ngafid.processor.steps.ComputeStep.required;
+
+
+public class ParquetFlightBuilder extends FlightBuilder {
+
+    public ParquetFlightBuilder(FlightMeta meta, Map<String, DoubleTimeSeries> doubleTimeSeries,
+                                 Map<String, StringTimeSeries> stringTimeSeries) {
+        super(meta, doubleTimeSeries, stringTimeSeries);
+    }
+
+    @Override
+    protected List<ComputeStep> gatherSteps(Connection connection) {
+        List<ComputeStep.Factory> parquetSteps = List.of(
+                required(ComputeStartEndTime::new),
+                ComputeAltAGL::new,
+                ComputeAirportProximity::new
+                //Skipping steps: ComputeLaggedAltMSL::new, ComputeTurnToFinal::new, ComputeItinerary::new, ComputeLOCI::new, ComputeTotalFuel::new, ComputeDivergence::new,  ComputeStallIndex::new,
+        );
+
+        ArrayList<ComputeStep> steps =
+                parquetSteps.stream()
+                        .map(factory -> factory.create(connection, this))
+                        .filter(step ->
+                                step.getOutputColumns().stream()
+                                        .noneMatch(x ->
+                                                getDoubleTimeSeriesMap().containsKey(x) ||
+                                                        getStringTimeSeriesMap().containsKey(x)
+                                        )
+                        )
+                        .collect(Collectors.toCollection(ArrayList::new));
```

Comment:
I don't think this is necessary anymore? And if it is, it would only apply to AltAGL in which case I recommend you just add an if statement.